homework/tests/homework.spec.js

```diff
@@ -0,0 +1,93 @@
+// @ts-check
+const { test, expect } = require('@playwright/test');
+
+test.describe('Testing each build', () => {
+     test('Build 1: An error message should be visible when letters are entered', async ({ page }) => {
+      await page.goto('https://testsheepnz.github.io/BasicCalculator');
```

Comment:
This is a common step for all tests. You could use a `beforeEach` hook before every test case and put this navigation to URL step there.